doc/filters/slice.rst

```diff
@@ -54,6 +54,12 @@ negative then the sequence will stop that many elements from the end of the
 variable. If it is omitted, then the sequence will have everything from offset
 up until the end of the variable.
 
+Core filters `first` and `last` are implemented using `slice`. If you need to get a specific element, use this:
+
+.. code-block:: jinja
+
+    {{ ['a', 'b', 'c', 'd'][-1:][0] }} {# will display "c" #}
```

Comment:
This is wrong, it will display `d`. But in this case, just use `{{ ['a', 'b', 'c', 'd'][-1] }}`.